src/Pact/Analyze/Translate.hs

```diff
@@ -1638,7 +1691,7 @@ runTranslation modName funName info caps pactArgs body checkType = do
     runBodyTranslation
       :: [Arg]
       -> VarId
-      -> Except TranslateFailure (ETerm, ([VarId], ExecutionGraph))
+      -> Except TranslateFailure (ETerm, ([VarId], ExecutionGraph, [TranslateFailure]))
```

Comment:
Should probably mark the `[TranslationFailure]` as `[TranslationWarnings]` or something similar

Reply:
We have future work for the output in general, we need to resolve to make typechecking and verification results errors and decide on a general warning scheme for both (using `validation` as you say or something else). For now this allows me to leverage the output utilities already present.

Reply:
Cool, makes sense.